ansible/playbooks/adhoc/uninstall.yml

```diff
@@ -0,0 +1,246 @@
+# This deletes *ALL* Origin, Atomic Enterprise Platform and OpenShift
```

Comment:
This file should stay on openshift-ansible, it is referenced in the official docs.